mod-source-record-manager-server/src/main/resources/templates/db_scripts/create_get_job_execution_summary_function.sql

```diff
@@ -39,10 +39,10 @@ BEGIN
       COUNT(*) FILTER (WHERE entity_type = 'AUTHORITY' AND (action_type = 'NON_MATCH' OR action_status = 'ERROR')) AS total_discarded_authorities,
       COUNT(*) FILTER (WHERE entity_type = 'AUTHORITY' AND action_status = 'ERROR') AS total_authorities_errors,
 
-      COUNT(*) FILTER (WHERE entity_type = 'ORDER' AND action_type = 'CREATE' AND action_status = 'COMPLETED') AS total_created_orders,
-      COUNT(*) FILTER (WHERE entity_type = 'ORDER' AND action_type = 'UPDATE' AND action_status = 'COMPLETED') AS total_updated_orders,
-      COUNT(*) FILTER (WHERE entity_type = 'ORDER' AND (action_type = 'NON_MATCH' OR action_status = 'ERROR')) AS total_discarded_orders,
-      COUNT(*) FILTER (WHERE entity_type = 'ORDER' AND action_status = 'ERROR') AS total_orders_errors,
+      COUNT(DISTINCT(order_id)) FILTER (WHERE entity_type = 'PO_LINE' AND action_type = 'CREATE' AND action_status = 'COMPLETED') AS total_created_orders,
```

Comment:
What is the number of rows we would expect average usage to be? 10s, 100s or 1000s? Thinking if the `DISTINCT` would hurt performance.

Reply:
Hello @okolawole-ebsco , it turned out that we do not need to return count of PO_LINE grouped by orderId, just return count of all PO_LINE, so I removed _DISTINCT_ at all https://github.com/folio-org/mod-source-record-manager/pull/746. Thank you!

Reply:
sounds good!